{
  checkThread();
  try {
    Assert.assertNull(message,object);
  }
 catch (  AssertionError e) {
    handleThrowable(e);
  }
}
